runtime/libpgmath/lib/CMakeLists.txt

```diff
@@ -235,23 +235,19 @@ endif()
 # Create libmath static library
 get_property(TARGET_OBJECTS GLOBAL PROPERTY "TARGET_OBJECTS")
 
-add_library(${LIBPGMATH_LIBRARY_NAME}_static STATIC ${TARGET_OBJECTS})
-add_library(${LIBPGMATH_LIBRARY_NAME} SHARED ${TARGET_OBJECTS})
-target_link_libraries(${LIBPGMATH_LIBRARY_NAME} INTERFACE ${LIBPGMATH_LIBRARY_NAME}_static)
-if(${LIBPGMATH_SYSTEM_NAME} MATCHES "Windows")
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME}_static PROPERTIES OUTPUT_NAME lib${LIBPGMATH_LIBRARY_NAME})
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME} PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
-if(${CMAKE_GENERATOR} MATCHES "MSYS Makefiles")
-add_custom_command (TARGET ${LIBPGMATH_LIBRARY_NAME} PRE_LINK
-    COMMAND  sed 's/ /\\n/g' CMakeFiles/pgmath.dir/objects1.rsp > objects1.txt
-    COMMAND  ${CMAKE_COMMAND} -E __create_def libpgmath.def objects1.txt
-)
-endif()
+add_library(pgmath_shared SHARED ${TARGET_OBJECTS})
+add_library(pgmath_static STATIC ${TARGET_OBJECTS})
 
-else()
-    set_target_properties(${LIBPGMATH_LIBRARY_NAME}_static PROPERTIES OUTPUT_NAME ${LIBPGMATH_LIBRARY_NAME})
-endif()
-install(TARGETS ${LIBPGMATH_LIBRARY_NAME}
-  LIBRARY DESTINATION lib)
-install(TARGETS ${LIBPGMATH_LIBRARY_NAME}_static
-  ARCHIVE DESTINATION lib)
+target_link_libraries(pgmath_shared INTERFACE pgmath_static)
+set_target_properties(pgmath_shared PROPERTIES OUTPUT_NAME pgmath WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+set_target_properties(pgmath_static PROPERTIES OUTPUT_NAME ${_lib}pgmath)
+
+install(TARGETS pgmath_shared
+        LIBRARY DESTINATION ${LLVM_LIBRARY_DIR}
+        ARCHIVE DESTINATION ${LLVM_LIBRARY_DIR}
+        RUNTIME DESTINATION ${LLVM_TOOLS_BINARY_DIR})
+
+install(TARGETS pgmath_static
+        LIBRARY DESTINATION ${LLVM_LIBRARY_DIR}
+        ARCHIVE DESTINATION ${LLVM_LIBRARY_DIR}
+        RUNTIME DESTINATION ${LLVM_TOOLS_BINARY_DIR})
```

Comment:
These variables (`LLVM_LIBRARY_DIR`, etc.) are defined in flang/CMakeLists.txt but I don't see how they get introduced into runtime/libpgmath/lib/CMakeLists.txt. If `CMAKE_INSTALL_PREFIX` is set properly, the original hardcoded destination (`lib`) would seem to work fine, wouldn't it?

Reply:
it would work fine, except in the case that these variables were changed.

Reply:
If you want, I can revert this.